sklearn/utils/validation.py

```diff
@@ -2270,9 +2270,9 @@ def _check_pos_label_consistency(pos_label, y_true):
             or np.array_equal(classes, [1])
         )
     ):
-        classes_repr = ", ".join(repr(c) for c in classes)
+        classes_repr = str(classes.tolist()).replace('[', '{').replace(']', '}')
```

Comment:
I kept the same error message as previously, but honestly I would be fine with using square brackets rather than curly brackets to simplify the code, i.e.: 
```
y_true takes value in ['a', 'b'] ...
```
rather than
```
y_true takes value in {'a', 'b'} ...
```